docs/pages/admin-guides/teleport-policy/crown-jewels.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: See permission changes with Access Graph Crown Jewels
-description: Describes how to use Access Graph Crown Jewels to see permission changes in Teleport.
+title: See permission changes with Graph Explorer Crown Jewels
+description: Describes how to use Graph Explorer Crown Jewels to see permission changes in Teleport.
 ---
 
-Access Graph's Crown Jewel feature allows you to track changes to access for
+Graph Explorer's Crown Jewel feature allows you to track changes to access for
```

Comment:
I've been splitting this out into another feature, and calling it `Teleport Policy Crown Jewels`

Reply:
Thank you, @benarent! I'll update this and the other references when I get back from vacation, 12/30.